Compiler/BackEnd/EvaluateParameter.mo

```diff
@@ -892,7 +908,21 @@ algorithm
       // apply replacements
       (e, true) = BackendVarTransform.replaceExp(e, iReplEvaluate, NONE());
       (e, _) = ExpressionSimplify.simplify(e);
-      e = EvaluateFunctions.evaluateConstantFunctionCallExp(e, FCore.getFunctionTree(iCache), Flags.getConfigBool(Flags.EVAL_CONST_ARGS_ONLY));
+      // If call with constant arguments then evaluate
```

Comment:
Why did you changed this? Ceval.ceval is very slow and cannot evaluate so much as ecaluateConstantFunctionCallExp can.